ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -304,6 +304,9 @@ if get_option('enable-cppfilter')
 endif
 
 if get_option('enable-edgetpu')
+  if not get_option('enable-filter-cpp-class')
+    error ('enable-filter-cpp-class should be set as \'true\' to build the tensor filter for Edge TPU.')
+  endif
   if get_option('enable-tensorflow-lite')
     edgetpu_dep = dependency('edgetpu', required: false)
     if not edgetpu_dep.found()
```

Comment:
LGTM 👍🏻👍🏻 Thanks for the contribution! I am writing a filter using C++ class after you.
One minor question: In line 314, doesn't need to `cc.find_library(...)` -> `cxx.find_library(...)` ?

Reply:
Yes, that'll be more accurate. As this commit is not touching find_library usages, you may create another PR to do it. Anyway, there many cases where find_library for C++ code is applied to cc, not cxx.